module/data/constants.py

```diff
@@ -68,3 +68,9 @@
 
 • RESIDENZA SAN MARZANO
 📞 095330868"""
+
+
+MENU_MENSA = 'Menù mensa 🍽' 
+CONTACT_ERSU = 'Contatti ERSU 📚'
+REPORT = 'Segnalazioni Rappresentanti 📬'
+HELP = 'help ❔'
```

Comment:
```suggestion
HELP = 'Help ❔'
```

Reply:
for back-compatibility with the previous bot it is important to keep the character case identical